packages/blocks/src/components/organisms/Sidebar/Sidebar.tsx

```diff
@@ -116,9 +116,10 @@ const SidebarItem: React.FC<
   compact = true,
   subNav,
   level,
+  defaultOpen,
```

Comment:
@german-mergestat by adding it here, it should address the message you were seeing in the console before

Reply:
ok let me release a new version and test it from `mergestat`.

Reply:
yep it worked... I did the change in this [PR](https://github.com/mergestat/blocks/pull/144/files). I think approving that PR will close this one.